idb/postgres/postgres_migrations.go

```diff
@@ -538,3 +528,56 @@ func MakeDeletedNotNullMigration(db *IndexerDb, state *MigrationState) error {
 	}
 	return sqlMigration(db, state, queries)
 }
+
+// MaxRoundAccountedMigration converts the import state.
+func MaxRoundAccountedMigration(db *IndexerDb, migrationState *MigrationState) error {
+	db.accountingLock.Lock()
+	defer db.accountingLock.Unlock()
+
+	nextMigrationState := *migrationState
+	nextMigrationState.NextMigration++
+
+	f := func(ctx context.Context, tx *sql.Tx) error {
+		defer tx.Rollback()
+
+		importstate, err := db.getImportState(tx)
+		if err == idb.ErrorNotInitialized {
+			// Leave uninitialized.
+			db.log.Printf("Import state is not initialized, leaving unchanged.")
+		} else if err != nil {
+			return err
+		} else {
+			if importstate.AccountRound == nil {
+				db.log.Printf("Account round is not set, leaving unchanged.")
+			} else {
+				nextRound := uint64(0)
+				if *importstate.AccountRound > 0 {
+					nextRound = uint64(*importstate.AccountRound + 1)
+				}
+				importstate.NextRoundToAccount = &nextRound
+				importstate.AccountRound = nil
+
+				db.log.Printf("Setting import state to %s", encoding.EncodeJSON(importstate))
+
+				err = db.setImportState(tx, importstate)
+				if err != nil {
+					return err
+				}
+			}
+		}
+
+		err = upsertMigrationState(db, tx, &nextMigrationState, false)
+		if err != nil {
+			return err
+		}
+
+		return tx.Commit()
```

Comment:
The upsert/commit can move into the else block, they're only needed when the `NextRoundToAccount` is actually changed/initialized.

Reply:
No, we need to increment the migration counter in all cases.